.github/workflows/npm-release.yml

```diff
@@ -0,0 +1,38 @@
+name: Publish on NPM
+
+on:
+  workflow_dispatch:
+    inputs:
+      version_type:
```

Comment:
It seems you expect as input a specific version and not a version type (patch,minor,major). Is it expected? Why not use incremental versioning, considering versions cannot be republished on npm?

Reply:
This is written in the PR description: 've not defined an auto versioning not to have to allow the action to push on main the bumped version (security reason).

Autoincrement is based on the version provided inside of the package.json, so the CI need to push the bumped version on main, which is not a good thing for a critical repo like this one IMO.